src/components/Header/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styles from './Header.styl';
+import HeaderPrimaryContent from '../HeaderPrimaryContent/HeaderPrimaryContent';
+import PageTitle from '../PageTitle/PageTitle';
+import HeaderTop from '../HeaderTop/HeaderTop';
+import FormRow from '../FormRow/FormRow';
+import TextField from '../TextField/TextField';
+import Button from '../Button/Button';
+import img from '../../img/header-bg';
+
+const Header = () => {
+  return (
+    <div className={styles.Header}>
+      <img src={img} className={styles.BgImg} />
```

Comment:
Just a note, there is no need to change... You can set this bg in css using some pseudo-elements (e.x. :before) and avoid loading here.